ldk/javascript/examples/self-test-loop/src/tests/screen/index.ts

```diff
@@ -109,3 +111,101 @@ export async function performOcr() {
       });
   });
 }
+
+console.log(`Starting app`);
+
+async function testOCRUsingCursor() {
+  let i = 0;
+  let cursorPositionStream: Cancellable;
+  let topParam: number;
+  let leftParam: number;
+  let topParam1: number;
+  let leftParam1: number;
+  await sleep(1000);
+  cursor
+    .listenPosition((response) => {
+      if (typeof response !== 'undefined') {
+        if (i === 0) {
+          topParam = response.y;
+          leftParam = response.x;
+          console.debug(`leftParam - X - ${leftParam1}`);
+          console.debug(` topParam - Y - ${topParam1}`);
+        }
+        if (i === 1) {
+          topParam1 = response.y;
+          leftParam1 = response.x;
+          console.debug(`leftParam1 - X - ${leftParam1}`);
+          console.debug(` topParam1 - Y - ${topParam1}`);
+        }
+
+        i += 1;
+
+        if (i >= 2) {
+          cursorPositionStream.cancel();
+        }
+      }
+    })
+    .then((cancellable: Cancellable) => {
```

Comment:
Since we're already doing async/await, can we await for the cancellable instead of promise chaining?